apps/amiapp_flutter/lib/src/screens/settings.dart

```diff
@@ -48,7 +49,7 @@ class _SettingsScreenState extends State<SettingsScreen> {
 
   @override
   void initState() {
-    widget._customerIOSDK.getDeviceToken().then((value) =>
+    CustomerIO.instance.pushMessaging.getRegisteredDeviceToken().then((value) =>
```

Comment:
In later PRs, I'll refactor this to remove `instance` requirement here and make `pushMessaging` static to align with API described in PR summary.